https://maplibre.org/about/ (#160)
maplibre/maplibre.github.io: content/about/duje/_index.md

---
title: About - Duje Schröder
weight: 10
---

<div class="text-center mb-5">
    <img
        src="https://avatars.githubusercontent.com/u/23352538?v=4"
        width="150"
        class="rounded-circle mt-3"
    />
    <h3 class="m-3">Duje Schröder</h3>
    <p>Assistant Coordinator</p>
</div>

<div class="text-center">
   _,--',     _,_._.--._______
 .--.--';_'-. ',_";_      _.,-'
.'--'.  _.'    {`'-;_ .-...'
      '-:_      )  / `' '=.
        ) >     {_/     /.)..
        |/               `..; .'
</div>

End? No, the journey doesn't end here. MapLibre is another path. One that we
all must take. On 2020-12-08 22:42:56 UTC MapLibre restarted mapping with its

> mission to provide freedom and independence. Eventually, this is what everybody

Reviewer: ```suggestion
mission to provide freedom and independence. In the end, this is what everybody
```

Author: https://dictionary.cambridge.org/dictionary/english/eventually 🤔

"In the end" sounds final as in at the end of a period, whereas "eventually" feels more like "some day (in the future)" and less opinionated or like a statement.

> is seeking, from a little project to enterprise endeavours. I am Duje and I am

Reviewer: ```suggestion
is seeking, from a little project to enterprise endeavours. I am Duje, and I am
```

assisting as a Coordinator to help maintain what the community has built so

> carefully which means to me:

Reviewer: ```suggestion
carefully. For me, this means:
```

- Assure MapLibre's fundraising and a healthy Sponsorship Program to keep
  MapLibre on the map

> - Listen to and discuss with the vivid community which corner to turn next

Reviewer: ```suggestion
- Listen to and discuss with the vibrant community which corner to turn next
```

English nitpick.

"vivid" usually is used with colors; "vibrant" is used with colors and people. :)

- Make MapLibre seen and accessible to everyone

> To sidewalk talks where everyone feels comfortable in the MapLibre world.

Reviewer: ```suggestion
Here's to sidewalk talks where everyone feels comfortable in the MapLibre world.
```

It took me a couple reads to realize what you meant. :smile:

> Gladly find me in my alley: <span><a

Reviewer: I'd rephrase this, as well. Something like "You can always find me on Slack". This phrase is a bit awkward in English.

Reviewer 2: @duje I will leave this one up to you. Let's merge it like this and if you want to update it, feel free to submit a follow-up pull request.

Author: Let it be a nice alley that one would find in Dubrovnik, Lisbon or Zurich 🙂
